bind_paths_and_mounts.rst

```diff
@@ -193,7 +193,7 @@ Mount specifications are also read from then environment variable
 ``$SINGULARITY_MOUNT``. Multiple bind mounts set via this environment
 variable should be separated by newlines (``\n``).
 
-Using ``--bind`` or ``-mount`` with the ``--writable`` flag
+Using ``--bind`` or ``--mount`` with the ``--writable`` flag
 ===========================================================
```

Comment:
The underline needs to be extended also to avoid RST lint errors.

Reply:
Sure thing. BTW, was the error supposed to show up in the check_rst workflow? If so, I'm not sure where to look: https://app.circleci.com/pipelines/github/sylabs/singularity-userdocs/156/workflows/425fdde4-aef9-4ebe-a007-b79f118e856a/jobs/522